src/Api/Billing/Controllers/StripeController.cs

```diff
@@ -46,4 +48,49 @@ public async Task<Ok<string>> CreateSetupIntentForCardAsync()
 
         return TypedResults.Ok(setupIntent.ClientSecret);
     }
+
+    [HttpPost]
+    [Route("~/tax/calculate")]
+    public async Task<IResult> CalculateAsync([FromBody] CalculateTaxRequestModel requestBody)
+    {
+        var options = new Stripe.Tax.CalculationCreateOptions
```

Comment:
We may need to wait on this PR until we get some clarity, but Stripe charges a fee per request when calculating tax. The fee is variable, and it might already be included in our plan, but we need to make sure we understand this first before we merge this. https://stripe.com/tax/pricing